examples/simple_arm_motion/README.md

```diff
@@ -0,0 +1,141 @@
+        robot.logger.info("Robot powered on.")
+
+        # Tell the robot to stand up. The command service is used to issue commands to a robot.
+        # The set of valid commands for a robot depends on hardware configuration. See
+        # RobotCommandBuilder for more detailed examples on command building. The robot
+        # command service requires timesync between the robot and the client.
+        robot.logger.info("Commanding robot to stand...")
+        command_client = robot.ensure_client(RobotCommandClient.default_service_name)
+        blocking_stand(command_client, timeout_sec=10)
+        robot.logger.info("Robot standing.")
+```
+The ROS2 code uses service calls to do the same thing:
+```python
+    # Claim robot
+    node.get_logger().info('Claiming robot')
+    result = robot.command('claim')
+    if not result.success:
+        node.get_logger().error('Unable to claim robot message was ' + result.message)
+        return False
+    node.get_logger().info('Claimed robot')
+
+    # Stand the robot up.
+    node.get_logger().info('Powering robot on')
+    result = robot.command('power_on')
+    if not result.success:
+        node.get_logger().error('Unable to power on robot message was ' + result.message)
+        return False
+    node.get_logger().info('Standing robot up')
+    result = robot.command('stand')
+    if not result.success:
+        node.get_logger().error('Robot did not stand message was ' + result.message)
+        return False
+    node.get_logger().info('Successfully stood up.')
+```
+Note that we also use ROS's text logger instead of the Spot API logger.
+
+Once the robot is initialized and standing, the next change to the code comes when we need to get the robot's current position.  With direct API calls, this uses the robot state:
+```python
+        robot_state = robot_state_client.get_robot_state()
+        odom_T_flat_body = get_a_tform_b(robot_state.kinematic_state.transforms_snapshot,
+                                         ODOM_FRAME_NAME, GRAV_ALIGNED_BODY_FRAME_NAME)
+```
+With ROS2, we use TF:
+```python
+    odom_T_flat_body = tf_listener.lookup_a_tform_b(ODOM_FRAME_NAME, GRAV_ALIGNED_BODY_FRAME_NAME)
+```
+
+The commands are built the same way after this, but we change how we send them.  The Spot API code has a function for waiting until the arm has moved to position:
+```python
+        # Send the request
+        cmd_id = command_client.robot_command(command)
+        robot.logger.info('Moving arm to position 1.')
+
+        # Wait until the arm arrives at the goal.
+        block_until_arm_arrives_with_prints(robot, command_client, cmd_id)
+```
+In ROS2, we convert the created protobuf to a ROS2 action goal and we use the action client `send_goal_and_wait` function to replace the `block_until_arm_arrives_with_prints` function (we do not get the printing, but we could echo the action feedback topic or use the non-blocking `send_goal_async` if we wanted to do that):
```

Comment:
"action client wrapper", so if someone skips down to here they aren't confused with the ROS2 ActionClient not having that function